todo_app/data/trello_items.py

```diff
@@ -0,0 +1,84 @@
+# This code uses the 'requests' library to call trello API
+# http://docs.python-requests.org
+import requests
+import os
+
+list_ids = {"to_do":"6501688a993cb949a6478d07",
+            "doing":"6501688bc072b227f990762d",
+            "done":"6501688a1c964b99f5ea6d20"
+}
+
+
+class TrelloManager:
+  
+    board_id = os.environ.get('BOARD_ID')
+    APIKey = os.environ.get('TRELLO_API_KEY')
+    APIToken = os.environ.get('TRELLO_TOKEN')
+    url = f"https://api.trello.com/1/boards/{board_id}/lists"
+
+
+    @classmethod
+    def get_trello_cards(cls):
+        """
+        function used to list all trello cards
+        """
+    
+        url = f"https://api.trello.com/1/boards/{TrelloManager.board_id}/lists"
+
+        query = {
+        'key': TrelloManager.APIKey,
+        'token': TrelloManager.APIToken,
+        'cards':'open',
+        'card_fields':['id','name', 'idList']
+        }
+
+        response = requests.request("GET", url, params=query)
+
+        response_json = response.json()
+        print(response_json)
```

Comment:
This is really helpful for debugging, but quite noisy for logs afterwards and importantly can be a source of data leak - it's very feasible that users of this app end up storing personal data here (like email addresses they want to contact, medical related to-dos or a thousand other sensitive things!) and we want to keep personal data out of our logs, so these can be good candidates to remove once we're happy the functions are working